riscv-page-tables/src/page_table.rs

```diff
@@ -283,11 +283,6 @@ impl<'a, T: PagingMode> PageTable<'a, T> {
         }
```

Comment:
Commit message needs some work. Good commit messages should have prose describing the change being made and why, along with possibly highlighting any particularly tricky or notable implementation details.

Reply:
Getting better, though "huge page support" is still overly generic. Something like "riscv-page-tables: Add support for mapping huge pages" would be more appropriate.